api/v1alpha3/cluster_types.go

```diff
@@ -91,13 +90,6 @@ type NetworkRanges struct {
 	CIDRBlocks []string `json:"cidrBlocks"`
 }
 
-func (n *NetworkRanges) String() string {
```

Comment:
This is a breaking change. 
What about replacing "" with "," here?

Reply:
As I said in the description I thought that the string representation is a Kubeadm-specific implementation detail, a comma-delimited list might make sense for Kubeadm but what if some other provider wants semicolons? But ultimately after I realized what the Kubeadm provider's `equality` package is doing I decided it was probably easier just to change the String method.